next-data/generators/websiteFeeds.mjs

```diff
@@ -2,18 +2,21 @@
 
 import { Feed } from 'feed';
 
-import { BASE_URL, BASE_PATH } from '../next.constants.mjs';
-import { siteConfig } from '../next.json.mjs';
+import { BASE_URL, BASE_PATH } from '../../next.constants.mjs';
+import { siteConfig } from '../../next.json.mjs';
```

Comment:
these should use our alias, right?

Reply:
Alias you mean `@/`? No we have that only configured for TypeScript.